packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -448,63 +448,63 @@ export const guides = [
         title: "Authorization",
         subLinks: [
           {
-            title: "Implement authorization by grant type",
+            title: "Implement authz by grant type",
             guideName: "implement-grant-type",
           },
           {
-            title: "Create an Authorization Server",
+            title: "Create an authorization server",
             guideName: "customize-authz-server",
           },
           {
-            title: "Configure SSO for Native apps",
+            title: "Configure SSO for native apps",
             guideName: "configure-native-sso",
           },
           { title: "Request user consent", guideName: "request-user-consent" },
           {
-            title: "Configure Device Authorization Grant Flow",
+            title: "Configure the Device Authz grant flow",
```

Comment:
Similar comment for authorization. And how about removing "the" as it's not really needed (these aren't proper sentences or even proper sentence fragments... just ways to help guide or remind the developer)

```suggestion
            title: "Configure Device grant flow",
```

Reply:
I would rather leave "Authz" in there because that is the actual name of the grant type flow - Device Authorization grant type. It fits - even on my laptop with an even more reduced size browser.